app/controllers/admin/field_groups_controller.rb

```diff
@@ -64,7 +63,7 @@ def sort
     field_group_ids = params["#{asset}_field_groups"]
 
     field_group_ids.each_with_index do |id, index|
-      FieldGroup.update_all({position: index+1}, {id: id})
+      FieldGroup.update_all({ position: index + 1 }, id: id)
```

Comment:
oops just spotted a bug here should be FieldGroup.where(id: id).update_all(position: index + 1)